sources/assets/exegol/load_supported_setups.sh

```diff
@@ -162,49 +166,34 @@ function deploy_python3() {
 }
 
 function run_user_setup() {
-  colorecho "Executing user setup"
+  timecolorecho "Executing user setup"
   # Executing user setup (or create the file)
   if [[ -f "$MY_SETUP_PATH/load_user_setup.sh" ]]; then
-    colorecho "[$(date +'%d-%m-%Y_%H-%M-%S')] ==== Loading user setup ($MY_SETUP_PATH/load_user_setup.sh) ===="
-    colorecho "Installing my-resources user's defined custom setup ..."
+    timecolorecho "==== Loading user setup ($MY_SETUP_PATH/load_user_setup.sh) ===="
+    timecolorecho "Installing my-resources user's defined custom setup ..."
     "$MY_SETUP_PATH"/load_user_setup.sh
   else
-    colorecho "[$(date +'%d-%m-%Y_%H-%M-%S')] ==== User setup loader missing, deploying it ($MY_SETUP_PATH/load_user_setup.sh) ===="
+    timecolorecho "==== User setup loader missing, deploying it ($MY_SETUP_PATH/load_user_setup.sh) ===="
     cp /.exegol/skel/load_user_setup.sh "$MY_SETUP_PATH/load_user_setup.sh"
     chmod 760 "$MY_SETUP_PATH/load_user_setup.sh"
   fi
-  colorecho "[$(date +'%d-%m-%Y_%H-%M-%S')] ==== End of custom setups loading ===="
+  timecolorecho "==== End of custom setups loading ===="
 }
 
-function deploy_firefox_addons() {
-  colorecho "Deploying Firefox Add-Ons"
-  local addon_folder
-  local addon_list
-  if [[ -d "$MY_SETUP_PATH/firefox/" ]]; then
-    if [[ -d "$MY_SETUP_PATH/firefox/addons" ]]; then
-      addon_folder="$MY_SETUP_PATH/firefox/addons"
-    else
-      mkdir "$MY_SETUP_PATH/firefox/addons" && chmod 770 "$MY_SETUP_PATH/firefox/addons"
-    fi
-    if [[ -f "$MY_SETUP_PATH/firefox/addons.txt" ]]; then
-      addon_list="$MY_SETUP_PATH/firefox/addons.txt"
-    else
-      cp --preserve=mode /.exegol/skel/firefox/addons.txt "$MY_SETUP_PATH/firefox/addons.txt"
-    fi
-    python3 /opt/tools/firefox/user-setup.py -L "$addon_list" -D "$addon_folder"
-  else
-    mkdir --parents "$MY_SETUP_PATH/firefox/addons" && chmod 770 -R "$MY_SETUP_PATH/firefox/addons"
-    cp --preserve=mode /.exegol/skel/firefox/addons.txt "$MY_SETUP_PATH/firefox/addons.txt"
+function deploy_firefox_policy() {
+  timecolorecho "Deploying Firefox Policy"
+  if [[ -f "$MY_SETUP_PATH/firefox/policies.json" ]]; then
+    cp --preserve=mode "$MY_SETUP_PATH/firefox/policies.json" /usr/lib/firefox-esr/distribution/policies.json
```

Comment:
The exegol-docs must be updated accordingly.

Reply:
already is https://github.com/ThePorgs/Exegol-docs/pull/73